src/ConnectedMode.UnitTests/Suppressions/TimedUpdateHandlerTests.cs

```diff
@@ -38,7 +39,8 @@ public void MefCtor_CheckIsExported()
                 MefTestHelpers.CreateExport<ISuppressionIssueStoreUpdater>(),
                 MefTestHelpers.CreateExport<IServerHotspotStoreUpdater>(),
                 MefTestHelpers.CreateExport<IQualityProfileUpdater>(),
-                MefTestHelpers.CreateExport<ILogger>());
+                MefTestHelpers.CreateExport<ILogger>(),
+                MefTestHelpers.CreateExport<IActiveSolutionBoundTracker>());
```

Comment:
Miscellaneous: could you add a "check is MEF singleton" test please?

Whenever I've touched an existing test file recently I've been checking it contains both of the "standard" MEF tests ("check is exported" and "check is/is not singleton") and adding them if necessary.

Reply:
The test is right below this one haha